src/components/ebay-skeleton/skeleton.stories.ts

```diff
@@ -83,7 +85,7 @@ export default {
 export const Default = Template.bind({});
 Default.args = {
     style: "width: 220px",
-    renderBody: `<div class="skeleton__textbox" />` as any,
+    renderBody: `<div role="img" aria-label="Loading" class="skeleton__textbox" />` as any,
```

Comment:
Is this needed still?

Reply:
Good catch, I did not see the wrapper being passed.